libs/application/data-providers/src/providers/CompanyRegistryProvider.ts

```diff
@@ -0,0 +1,46 @@
+import { coreErrorMessages } from '@island.is/application/core'
+import {
+  BasicDataProvider,
+  Application,
+  SuccessfulDataProviderResult,
+  FailedDataProviderResult,
+} from '@island.is/application/types'
+import { RskCompany } from '@island.is/api/schema'
+
+export class CompanyRegistryProvider extends BasicDataProvider {
```

Comment:
This provider comes unnecessary as the new Identity Provider gives you the company name, and should be use instead.